test/any/confluence/api_count-test.es6.js

```diff
@@ -38,30 +41,29 @@ describe('ApiCount', () => {
       ],
     }, container);
     releases = container.releaseDAO;
-    ifaces = container.webInterfaceDAO;
-    junctions = container.releaseWebInterfaceJunctionDAO;
-  });
+    compatData = container.compatDAO;
+
+    return releaseSpecs.map(rs => Release.create(rs, container));
+  };
 
   it('should compute single release', done => {
-    let release = Release.create({
+    let release = {
       browserName: 'Alpha',
       browserVersion: '1',
       osName: 'Windows',
       osVersion: '10',
       releaseDate: '2015-01-01T00:00:00.000Z'
-    }, container);
-    let iface = WebInterface.create({
+    };
+    [release] = init([release]);
```

Comment:
What does this syntax do?

Reply:
It unpacks the array returned by `init(...)`, expecting that its length is at least `1`, and assigns the first element to the variable named `release`. The intent is to elevate the release spec passed in before code generation to a "real" `Release` object.